src/readonly-table/index.ts

```diff
@@ -158,7 +158,15 @@ let exportReadOnlyTable;
         >
           ${this._columnOrder.map((currentCol, colIndex) => {
             const visible = this.columns[colIndex].visible ?? true;
-            const value = data[currentCol];
+            let value = data[currentCol];
+
+            if (this._isHTML(value)) {
+              value = html`<div
+                class="kuc-readonly-table__table__body__row__cell-data--html"
```

Comment:
Why do we need to add an extra 'div' with the class `kuc-readonly-table__table__body__row__cell-data--html` to wrap the HTML here?

Reply:
Because there is a CSS setting `white-space: pre-wrap` in `.kuc-readonly-table__table__body__row__cell-data`.
https://github.com/kintone-labs/kintone-ui-component/blob/9dbc56eba1f4df36de8f8911293b2958f86db040/src/readonly-table/style.ts#L73C4-L73C27
This setting creates some unnecessary line breaks in the cell when using the KUC components in it.

So I added `white-space: normal;` in a class `kuc-readonly-table__table__body__row__cell-data--html` to override it when using HTMLElement.

Reply:
I got it.
Thank you!